objects/obj_boss_sphere/Destroy_0.gml

```diff
@@ -14,4 +14,8 @@ instance_create_layer(x, y, "enemies", obj_explosion);
 //create coins
 with instance_create_depth(x,y,depth-1,obj_coin_spawner) {
 	num_of_coins = 100;
+}
+// set active when destroy
+with(obj_boss_portal) {
```

Comment:
Currently, when this code get's reached there's no obj_boss_portal in the boss room. Do you plan to create that object here or are you going to place one in the room?